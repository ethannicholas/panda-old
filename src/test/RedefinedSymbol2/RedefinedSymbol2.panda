class RedefinedSymbol2 {
    var length:Int := 5

    function length():Int {
        return length
    }
}
