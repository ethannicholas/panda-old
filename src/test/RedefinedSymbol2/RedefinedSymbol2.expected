error: RedefinedSymbol2.panda:4:5: invalid redefinition of symbol 'length'
1 error
